src/mmw/js/src/core/views.js

```diff
@@ -154,11 +149,36 @@ var MapView = Marionette.ItemView.extend({
 
         map.addControl(new L.Control.Zoom({position: 'topright'}));
         addLocateMeButton(map, maxZoom, maxAge);
-        map.addLayer(tileLayer);
         map.addLayer(areaOfInterestLayer);
         map.addLayer(modificationsLayer);
         map.setView([40.1, -75.7], maxZoom); // center the map
 
+         $.get('/base-layers/')
+            .done(function(response) {
+                var baseLayers = _.mapObject(response, function(layerData) {
+                    return new L.TileLayer(layerData.url, {
+                        attribution: layerData.attribution || '',
+                        maxZoom: layerData.maxZoom || 18
+                    });
+                }),
+                    defaultLayerName = _.findKey(response, function(layerData) {
+                        return layerData.default;
+                    });
+
+                L.control.layers(baseLayers).addTo(map);
```

Comment:
Just tried it on the iPad, works well. Tap once to expand, then tap to select option. The expanded form stays open until you tap elsewhere.